src/Todos.js

```diff
@@ -0,0 +1,61 @@
+class Todos {
+  static todos = [];
+
+  constructor(id, description) {
+    this.id = id;
+    this.description = description;
+    this.completed = false;
+  }
+
+  static storage = (todos) => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
```

Comment:
Use the advantage of the anonymous function and as the code is just a single line the Flower Brackets can be removed.
 The suggestion is to use:
`static storage = (todos) => localStorage.setItem('todos', JSON.stringify(todos));`

Reply:
Done!